base/dates/parse.jl

```diff
@@ -110,59 +126,58 @@ function Base.parse(::Type{DateTime}, s::AbstractString, df::DateFormat{Symbol("
     th = tm = ts = tms = Int64(0)
 
     nv, i = tryparsenext_base10(s,i,end_pos,1)
-    dy = isnull(nv) ? (@goto error) : get(nv)
+    dy = isnull(nv) ? (@goto error) : nv.value
```

Comment:
I was unaware of `unsafe_get`. I'll update to use it and make sure there isn't any performance hit.